src/base/__tests__/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { TypeDef } from '@polkadot/types/types';
+
+import { TransactionArgumentType } from '~/types';
+
+import { processType } from '../utils';
+
+describe('Process Type', () => {
+  it('should be a function', () => {
+    expect(typeof processType).toBe('function');
```

Comment:
```suggestion
    expect(processType).toBeInstanceOf(Function)
```

(nit) I think built ins are a bit more readable